src/deepsparse/loggers_v2/filters/pattern.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+from typing import Optional
+
+
+def is_match_found(
```

Comment:
note that treating all targets/patterns as pure regex may have unintended side effects

for example, `.` is a wildcard, so `fooXbar` will also match with`foo.bar`

we might want to think through if we want to protect regex use a bit more